com/epam/brigada_5/java/lesson_1/task_2/Array.java

```diff
@@ -0,0 +1,38 @@
+package com.epam.brigada_5.java.lesson_1.task_2;
+import java.util.Scanner;
+
+public class Array {
+    String[] array;
+    int n;
+
+    public void arrayFilling() {
+        Scanner in = new Scanner(System.in);
+        System.out.println("Enter words quantity: ");
+        n = in.nextInt();
+        array = new String[n];
+
+        for (int i = 0; i < n; i++) {
+            System.out.println("Enter word №" +(i+1)+ ":");
+            array[i] = in.next();
+        }
+    }
+
+    public void arrayChecking() {
+        String word;
+        char[] chars;
+        int m = 0;
+        for (int i = 0; i < n; i++) {
+            word = array[i];
+            chars = word.toCharArray();
+            for (int j = 0; j < chars.length-1; j++) {
+                if (chars[j] != chars[j+1]) {
+                    m++;
+                }
+            }

```

Comment:
Если chars нигде не используется, то не надо зря расширять скоуп переменных
```suggestion
        int m = 0;
        for (int i = 0; i < n; i++) {
            String  word = array[i];
            char[] chars = word.toCharArray();
            for (int j = 0; j < chars.length-1; j++) {
                if (chars[j] != chars[j+1]) {
                    m++;
                }
            }
```